core/src/main/scala/com/yahoo/maha/core/FilterOperation.scala

```diff
@@ -772,6 +839,19 @@ object FilterDruid {
       case f @ EqualityFilter(alias, value, _, _) => {
         getDruidFilter(grainOption, column, columnAlias, value, columnsByNameMap)
       }
+      case f @ FieldEqualityFilter(field, compareTo, _, _) => {
+        require(aliasToNameMapFull.contains(field) && aliasToNameMapFull.contains(compareTo), s"A requested field is invalid!  Tried: $field & $compareTo")
```

Comment:
Same, we don't want to do validation here.  It should be done in request model.

Reply:
With this and the prior suggestion, the first piece of validation (original way) is being done by way of try-catch in the code block here.  I could attempt to move this validation into the requestModel completely.